src/sentry/web/frontend/user_avatar.py

```diff
@@ -31,4 +32,7 @@ def get(self, request, *args, **kwargs):
                 return HttpResponseBadRequest()
             else:
                 photo_file = avatar.get_cached_photo(size)
-        return HttpResponse(photo_file, content_type='image/png')
+
+        res = HttpResponse(photo_file, content_type='image/png')
+        patch_cache_control(res, max_age=315360000)
```

Comment:
Nah, avoid this. It provides us little value.

Checkout what I'm doing in `sentry.web.frontend.generic:static_media`.

There's a `CACHE_FOREVER` constant that could be shared here.